iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml

```
@@ -41,9 +12,8 @@ spec:
       {{- if .Values.capabilities }}
         capabilities:
           add:
-# Noncompliant@+1
+# Noncompliant@+2
           {{- range .Values.capabilities }}
-#                          ^^^^^^^^^^^^^^^^
             - {{ . | quote }}
```

```suggestion
            - {{ . | quote }}
            #^[sc=0;ec=29]
```
So now we will highlight this part, right?

---

Right. But I'm not a fan of this syntax and would have a clean loop without comments in between.
Unfortunately, this change introduced regression in such Helm loops, because now the issue is raised on `{{ . | quote }}` what doesn't contain any value path and the trailing comment with line numbers doesn't include the above line.
We have to think about how to solve such cases (detecting loops, looking before, etc.) but it's a topic for the next iteration.